.github/ISSUE_TEMPLATE/2-deprecate-sgid-dataset.md

```diff
@@ -30,15 +30,19 @@ The replaced data is still accessible via our shelved policy in AGOL (a link to
 1. _~Strike~ out all items that do not apply._
 
 - [ ] Remove data from the Internal SGID (name, completed: `2021/00/00`)
+- [ ] Mark ArcGIS Online item as deprecated in preparation for future deletion (name, completed: `2021/00/00`)
+  - [ ] Add `Deprecated` to `AGOL_PUBLISHED_NAME` in `SGID.META.AGOLItems` (name, completed: `2021/00/00`)
+  - [ ] Change `Authoritative` field to `d` in `SGID.META.AGOLItems` to set the `Deprecated` AGOL flag (name, completed: `2021/00/00`)
```

Comment:
**thought**: could we ignore the published name and if the field is set to `d` then auditor modifies the title in a consistent manner.

Reply:
I can investigate updating Auditor to automatically add `(Deprecated)` to the title if the field is set to `d`. That would get us around the table name issue.

Reply:
Agreed. If Auditor can add `Deprecated` to the agol title based on the META.AGOLItems field set to `d` that would be great.

Reply:
Auditor will now add `(Deprecated)` per https://github.com/agrc/auditor/pull/53